pom.xml

```diff
@@ -10,7 +10,9 @@
     <modules>
         <module>GameEngine</module>
         <module>AI</module>
+<!--
```

Comment:
Why did you out-comment the bullet module?

Reply:
my bad, glemte at jeg havde gjort det XD